draw-lib/racket/draw/private/bitmap-dc.rkt

```diff
@@ -186,6 +186,11 @@
            (when (eq? s 'unsmoothed) (set-smoothing 'unsmoothed))
            (set-transformation t)))))
 
+    (define/override (get-text-extent . args)
+      (if (internal-get-bitmap)
+          (super get-text-extent . args)
+          (send (get-temp-bitmap-dc) get-text-extent . args)))
+
```

Comment:
Using a rest argument here may interfere with reflection on the method's arity. I don't have a better idea than repeating the arguments and defaults, although the contracts can be omitted (since they'll be checked on the `super` call):

```
    (define/override (get-text-extent s
                                      [use-font (get-font)]
                                      [combine? #f]
                                      [offset 0])
      (if (internal-get-bitmap)
          (super get-text-extent s use-font combine? offset)
          (send (get-temp-bitmap-dc) get-text-extent s use-font combine? offset)))
```